dist/access_check/check_sources.py

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/env python3
+
+import sys, os
+from layercparse import *
+from pprint import pprint, pformat
+
+def main():
+    # setLogLevel(LogLevel.WARNING)
+
+    rootPath = os.path.realpath(sys.argv[1])
+    setRootPath(rootPath)
+    setModules([
+        Module("block"),
+        Module("block_cache", fileAliases=["block_chunkcache"], sourceAliases = ["blkcache", "bm"]),
+        Module("bloom"),
+        Module("btree", fileAliases=["btmem", "btree_cmp", "dhandle", "modify", "ref", "serial"]),
+        Module("call_log"),
+        # Module("checksum"),
+        Module("conf", sourceAliases=["conf_keys"]),
+        Module("config"),
+        Module("conn", fileAliases=["connection"], sourceAliases=["connection"]),
+        Module("cursor", sourceAliases=["cur", "btcur", "curbackup"]),
+        Module("evict", fileAliases=["cache"]),
+        Module("history", sourceAliases = ["hs"]),
+        Module("log"),
+        Module("lsm", sourceAliases=["clsm"]),
+        Module("meta", sourceAliases=["metadata"]),
+        Module("optrack"),
+        # Module("os", fileAliases = ["os_common", "os_darwin", "os_linux", "os_posix", "os_win"]),
+        Module("packing", sourceAliases=["pack"]),
+        Module("reconcile", sourceAliases = ["rec"]),
+        Module("rollback_to_stable", sourceAliases = ["rts"]),
+        Module("schema"),
+        Module("session"),
+        # Module("support"),
+        Module("tiered"),
+        Module("txn", sourceAliases=["truncate"]),
+        # Module("utilities"),
+
+        Module("bitstring"),
+        Module("cell"),
+        Module("checkpoint", sourceAliases=["ckpt"]),
+        Module("column", sourceAliases=["col"]),
+        Module("compact"),
+        Module("generation"),
+        Module("pack", fileAliases=["intpack"]),
+        Module("stat"),
```

Comment:
I can see you started by enumerating the "modules" and commenting on those non-modules. It should work well right now but will have some maintenance overhead to update this file later when modules (or non-modules) are added/deleted/restructured. 

Another approach is to loop through all sub-dirs under "src/" and have a skip list for non-modules. It likely won't be maintenance-free (e.g. when source and file aliases need to be specified/updated) but can save some effort?

Reply:
Those commented-out lines (for non-modules) deserve at least a comment (if not a skip list).

Reply:
Sounds good